R/fragments-calculate2.R

```diff
@@ -1,55 +1,183 @@
 #' @title Calculate ions produced by fragmentation with variable modifications
```

Comment:
This file should be removed completely, or?

Reply:
Yes, quite confusing, as they are nearly identical.